t/cli/test_kubernetes.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+. ./t/cli/common.sh
+
+exit_if_not_customed_nginx
```

Comment:
We can run this test with regular Nginx?

Reply:
Sorry, I don't quite get what you mean.

But I think the running environment of this test file should be the same as other test files in the same directory.

Reply:
> We can run this test with regular Nginx?

Sorry, I don't quite get what you mean.

But I think the running environment of this test file should be the same as other test files in the same directory.

Reply:
> > We can run this test with regular Nginx?
> 
> Sorry, I don't quite get what you mean.
> 
> But I think the running environment of this test file should be the same as other test files in the same directory.

I got ， and will fix